Sources/XCHammer/main.swift

```diff
@@ -184,18 +184,37 @@ func main() {
     commands.register(HelpCommand(registry: commands))
 
     var arguments = CommandLine.arguments
-    // Remove executable name
     arguments.remove(at: 0)
-
-    func handle(error: CommandError) {
-        print("------")
-        print("--- EXCEPTION ---")
-        print(error)
-        print(error.localizedDescription)
-        print("------")
+    // Hack to remove Xcode running arguments
+    arguments = arguments.reduce(into: [], {
+        result, next in
+        if next == "-NSDocumentRevisionsDebugMode" {
```

Comment:
Leave a comment around what we are removing and why

Reply:
I mentioned above - not sure why Xcode adds these ( probably for AppKit )
> // Hack to remove Xcode running arguments